internal/metric/smart_metrics.go

```
@@ -0,0 +1,201 @@
+	_, err := exec.LookPath("smartctl")
+	if err != nil {
+		return fmt.Errorf("smartctl is not installed")
+	}
+	return nil
+}
+
+// scanDevices returns a list of disks using `smartctl --scan`
+func scanDevices() ([]string, error) {
+	out, err := exec.Command("smartctl", "--scan").Output()
+	if err != nil {
+		return nil, fmt.Errorf("failed to scan devices: %v", err)
```

golangci-lint (1.64.8)

non-wrapping format verb for fmt.Errorf. Use `%w` to format errors

(errorlint)